spec/requests/api/v1/groups_spec.rb

```diff
@@ -106,5 +106,72 @@
         expect(response).to have_http_status(:ok)
       end
     end
+
+    describe 'POST /add_members' do
+      let(:group) { Group.create(name: "Example Group 1", description: "Sample Description") }
+      let(:user) { create(:user) }
+
+      context 'when group and user exist' do
+        before do
+          post :add_members, params: { id: group.id, user_id: user.id }
+        end
+
+        it 'adds the user to the group' do
+          expect(group.users).to include(user)
+        end
+
+        it 'returns a successful response with the updated group users' do
+          expect(response).to have_http_status(:ok)
+        end
+      end
+
+      context 'when group or user does not exist' do
```

Comment:
update the tests in such a way that you are testing for both group missing and user missing scenarios separately